### DISCUSSION TITLE
server error when opening ipam/ip-addresses/

### DISCUSSION THREAD

**Discussion by behnamyazdani** [netbox-community/netbox#12670] [Help Wanted!] [answered]
Hi
I get below server error whey I try to open `ipam/ip-addresses/` page
`https://<URL>/ipam/ip-addresses/`

- Searching IP addresses works
- Individual IP address URLs work with no issue example: `https://<URL>/ipam/ip-addresses/2/`
- No custom links are created 
- Other root menu works too: asn, ip-ranges, prefixes, ... 
- Calling `ip-addresses` via API works: `https://<URL>/api/ipam/ip-addresses/`

### OS:
Ubuntu 20.04 server

### Installation type: 
VM on esxi

#### Server Error

There was a problem with your request. Please contact an administrator.

------

The complete exception is provided below:

```
<class 'AttributeError'>

'NoneType' object has no attribute 'get_absolute_url'

Python version: 3.8.10
NetBox version: 3.5.1
```

**Comment by candlerb** [comment, marked as answer]
Search the archives: https://github.com/netbox-community/netbox/discussions?discussions_q=%27NoneType%27+object+has+no+attribute+%27get_absolute_url%27

This is a known problem where an IP Address gets into a state where it references a null object id but a non-null object type.

There is a custom script which should fix it for you here:
https://github.com/netbox-community/customizations/pull/85/files

**Reply by behnamyazdani** [reply]
Thank you! solved
